CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 `servant-hmac-auth` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+## 0.1.6 - TBD
```

Comment:
Let me know if you're fine with releasing the package as is and I'll bump the version in cabal file and upload it to hackage.